chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -57,36 +57,42 @@ data:
       # Ensure each cluster can be reached (should only be
       # used with an auth-proxy where k8s credentials never leave
       # the cluster). See clusters option.
-      {{- range .Values.clusters }}
+    {{- range .Values.clusters }}
       location ~* /api/clusters/{{ .name }} {
-        rewrite /api/clusters/{{ .name }}/(.*) /$1 break;
-        rewrite /api/clusters/{{ .name }} / break;
-
-      {{/* Helm returns a nil pointer error when accessing foo.bar if foo doesn't
+    {{/* We need to split the API service URL(s) into the base url and the path segment so
+           those configurations using a path can be appropriately rewritten below while
+           ensuring the proxy_pass statement is given the base URL only. */}}
+    {{- $parsed := urlParse (default "https://kubernetes.default" .apiServiceURL) }}
```

Comment:
I made the default value `https://kubernetes.default` here.

Reply:
Yep, it works there too. I was keen to leave it where it was for other reasons (I'll need the actual value set `""` to be passed to other services such as pinniped-proxy in this same file, so that they can work with what was actually set by the user and make decisions accordingly, rather than not knowing if the user explicitly set `https://kubernetes.default`). But I can update that if/when we ever need this.